web/src/test/java/org/springframework/security/web/authentication/logout/CookieClearingLogoutHandlerTests.java

```diff
@@ -59,7 +59,7 @@ public void configuredCookiesAreCleared() {
 		assertThat(response.getCookies()).hasSize(2);
 		for (Cookie c : response.getCookies()) {
 			// gh-2325
```

Comment:
Please remove this comment since the updated behaviour no longer aligns with what is mentioned in issue 2325.